### PR TITLE
Add sent and received requests

### PR DESCRIPTION
**Pull request by hidehiro98**
@db0sch 
Sorry for many commits in one PR, next time, I will make PR more oftenly.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @request = Request.find(params[:request_id])
+    @comment = Comment.new(comment_params)
+    @comment.request = @request
+    @comment.author = current_user
+    if @comment.save
+      redirect_to request_path(@request)
+    else
+      redirect_back(fallback_location: root_path)
```
Comment (db0sch) [inline app/controllers/comments_controller.rb]: if it doesn't save, you want to render the page where the form was submitted.
with this: 
```
flash[:alert] = "Information missing."
```

File: app/controllers/requests/received_requests_controller.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 class Requests::ReceivedRequestsController < ApplicationController
   def index
+    receivers = Receiver.where(user: current_user)
```
Comment (db0sch) [inline app/controllers/requests/received_requests_controller.rb]: why not something like this : `current_user.receivers`

File: app/controllers/requests/received_requests_controller.rb [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+    @requests = []
+    receivers.each { |receiver| @requests << receiver.request }
+    @requests
```
Comment (db0sch) [inline app/controllers/requests/received_requests_controller.rb]: but then, I guess you just need the requests. so we don't need the receivers.

so I would erase all that, and just keep:

```
@requests = current_user.requests
```

But then you need to define a scope, to get only the requests you received.
Mmmmh.

or maybe, in your model, you'll need something like:
```
has_many :received_requests, class_name: 'requests', through: :recievers

```

**Review by db0sch — APPROVED** [review]

File: app/controllers/requests/received_requests_controller.rb (hunk 3 above)
Comment (hidehiro98) [reply app/controllers/requests/received_requests_controller.rb]: Yes, I think the last choice (creating new association) is the best.  I'll make this association.

Comment (hidehiro98) [reply app/controllers/requests/received_requests_controller.rb]: `has_many :received_requests, through: :receivers, source: :request` is better in this case.
Cecile said so in the Le Wagon in this code.
https://github.com/hidehiro98/heytask/blob/master/app/models/user.rb

Cecile used `class_name` in this code.
https://github.com/hidehiro98/heytask/blob/master/app/models/service.rb


And I researched the difference between `class_name` and `source`.
https://stackoverflow.com/questions/13611265/rails-difference-between-source-and-class-name-in-models

**Pull request merged**